.github/workflows/copy-build-to-d2-ci.yml

```diff
@@ -0,0 +1,51 @@
+name: Copy build to d2-ci
+
+on:
+    push:
+        branches-ignore:
+            - master
+            - next
+            - next-major
+            - alpha
+            - beta
+            - '[0-9]+.x'
+            - '[0-9]+.x.x'
+            - '[0-9]+.[0-9]+.x'
+
+env:
+    GH_TOKEN: ${{secrets.DHIS2_BOT_GITHUB_TOKEN}}
+
+jobs:
+    copy-to-d2-ci:
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout code
+              uses: actions/checkout@v3
+              with:
+                  token: ${{env.GH_TOKEN}}
+
+            - name: Set up Node.js
+              uses: actions/setup-node@v1
+              with:
+                  node-version: '16.x'
+
+            - name: Install dependencies
+              run: yarn install --frozen-lockfile
+
+            - name: Build package
+              run: yarn build
+
+            - name: Pack and unpack the build to a directory named 'package'
+              run: yarn pack --filename output.tgz && tar -xzf output.tgz
+
+            - uses: actions/upload-artifact@v2
+              with:
+                  name: output-build
+                  path: output.tgz
+                  retention-days: 2
+
+            - name: Copy package to d2-ci
+              uses: dhis2/deploy-build@master
+              with:
+                  build-dir: package
+                  github-token: ${{env.GH_TOKEN}}
```

Comment:
I can't tell from this what the path to this artefact would be, but it would be convenient if:
1. The path/URL only has one dynamic section, which is the branch-name
2. We can have one artefact per branch (this would be the case if we implement point 1)